sonic-ledd/scripts/ledd

```diff
@@ -17,6 +17,7 @@ try:
     from sonic_daemon_base import daemon_base
     from sonic_daemon_base.daemon_base import Logger
     from sonic_daemon_base.daemon_base import DaemonBase
+    from sonic_daemon_base.daemon_base import multiAsic
```

Comment:
This might be for another PR, but if `multiAsic` is a class name, it should be in PascalCase (i.e., `MultiAsic`)

Reply:
Changed the name to MultiAsic and updated the other PR as well.